infrastructure/code/backend_service_test.go

```diff
@@ -166,3 +168,35 @@ func TestSnykCodeBackendService_RunAnalysisSmoke(t *testing.T) {
 
 // todo analysis test limit files
 // todo analysis test severities
+
+func TestGetCodeApiUrl(t *testing.T) {
+
+	t.Run("Default deeprox url for code api", func(t *testing.T) {
+		c := config.CurrentConfig()
+
+		url, _ := getCodeApiUrl(c)
+		assert.Equal(t, c.SnykCodeApi(), url)
+	})
+
+	t.Run("Fedramp url for code api", func(t *testing.T) {
+		c := config.CurrentConfig()
+		t.Setenv("DEEPROXY_API_URL", "")
+		c.UpdateApiEndpoints("https://app.snykgov.io")
+		orgUUID, _ := uuid.NewRandom()
+		c.SetOrganization(orgUUID.String())
+
+		url, err := getCodeApiUrl(c)
+		assert.Nil(t, err)
+		assert.Equal(t, "https://api.snykgov.io/hidden/orgs/"+orgUUID.String()+"/code", url)
+	})
+
+	t.Run("Fedramp url for code api", func(t *testing.T) {
```

Comment:
Suggestion: There are some legacy formats of the API Url, which might be worth testing for. Please take a look [here](https://github.com/snyk/go-application-framework/blob/main/internal/api/urls_test.go)